README.md

```diff
@@ -40,9 +40,9 @@ yarn install
 You'll have three terminals up for:
 
 ```bash
-yarn start   (Next app frontend)
 yarn chain   (hardhat backend)
 yarn deploy  (to compile, deploy, and publish your contracts to the frontend)
+yarn start   (Next app frontend)
```

Comment:
Hey! Check `base-challenge-template` branch. This block was changed to this: 


> in the same terminal, start your local network (a blockchain emulator in your computer):

```sh
yarn chain
```

> in a second terminal window, 🛰 deploy your contract (locally):

```sh
cd <challenge_folder_name>
yarn deploy
```

> in a third terminal window, start your 📱 frontend:

```sh
cd <challenge_folder_name>
yarn start
```

📱 Open http://localhost:3000 to see the app.

Reply:
I think the question is if they have already learned that from Challenge-0.  If not we can be as verbose as the base template instructions, but if they know it already we can condense it down.  Thoughts?

Reply:
I think it's not a big deal either way.

I'd personally choose to be as verbose as the base template:
- People might not do the challenges in order (we are also now pointing to specific challenges in the new BG website)
- Consistency

Reply:
Commit #3 adds the verbose instructions, with renumbered checkpoints.